helm/gadgetron/values.yaml

```diff
@@ -62,3 +51,25 @@ resources:
 nodeSelector: {}
 
 tolerations: []
+
+storageServer:
+  replicaCount: 1
+
+  image: ghcr.io/ismrmrd/mrd-storage-server:v0.0.8
+  imagePullPolicy: IfNotPresent
+
+  storageAccountConnectionStringSecretName: storageserversa
+  storageAccountConnectionStringSecretDataKey: connectionString
+
+  imagePullSecrets: []
+  podAnnotations: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  podSecurityContext: {}
+  securityContext: {}
+  resources: {}
+
+storagedb:
+  usePasswordFile: true
```

Comment:
Looks like this value is not used

Reply:
Interesting. Was looking at https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml#L146 and it appears to be under `auth`, but that also didn't seem to do much. I am removing it from here for now. But for other projects we should investigate. Also either removed or made function a few of the other parameters on the storage server (like resource requests and nodeSelection, which might be handy).

Reply:
I'm sorry, I think I was completely wrong about this. See https://github.com/bitnami/charts/blob/d080ba6aeb7dea0342101d7c2e487cfdbf509029/bitnami/postgresql/templates/primary/statefulset.yaml#L226 

I didn't realize this this value was from the postgresql subchart, I just thought it was an an unreferenced value.